docker-compose-devnet.yml

```diff
@@ -25,6 +25,7 @@ services:
     environment:
       - TEXLOTUSDEVNET_SPEED=500
       - TEXLOTUSDEVNET_IPFSADDR=/dns4/ipfs/tcp/5001
+      - TEXLOTUSDEVNET_BIGSECTORS=${BIGSECTORS}
```

Comment:
@jsign just want to be sure this is the right way to set the right env var. My goal here is that someone could launch docker compose in one of three ways:

1. `docker-compose -p devnet -f docker-compose-devnet.yml up --build -V`
2. `BIGSECTORS=true docker-compose -p devnet -f docker-compose-devnet.yml up --build -V`
3. `BIGSECTORS=false docker-compose -p devnet -f docker-compose-devnet.yml up --build -V`

Do you think those will all work with this setup? I'm assuming the unset `BIGSECTORS` will resolve to `false`.

Reply:
Ref: https://docs.docker.com/compose/environment-variables/#substitute-environment-variables-in-compose-files

Reply:
I don't think that the unset case will be interpreted correctly, since it I think would be the empty string. But maybe it may worth trying.

It may be safer to provide some `make bigsectors` and `make` to avoid improper configurations?

Reply:
Within this project, yes, but I point people to this compose file in the blog post tutorial as something to copy and use in the example project.

Reply:
I'll do a test to see how viper resolves the empty string, then we can go from there

Reply:
Yea looks like viper converts an string env var to false when you ask for it using getBool